Jenkinsfile

```diff
@@ -17,6 +17,7 @@
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
 //@Library(value='pipeline-lib@your_branch') _
+@Library(value='pipeline-lib@hendersp/DAOS-17537') _
```

Comment:
Obviously, we can't land this PR with this still in it.

Reply:
Removed in https://github.com/daos-stack/daos/pull/16224/commits/0172aa3193ecd77fa6c2788a2a1bf92820cb5138 after landing https://github.com/daos-stack/pipeline-lib/commit/efbd410ce8590818343054f342c246ac74e83ab3